logstash-core/lib/logstash/runner.rb

```diff
@@ -82,6 +82,10 @@ class LogStash::Runner < Clamp::Command
     I18n.t("logstash.runner.flag.auto_reload"),
     :attribute_name => :auto_reload, :default => false
 
+  option ["-ap", "--http-port"], "WEB_API_HTTP_PORT",
```

Comment:
I really don't like 2 character short flags...in many unix utilities that is equivalent to -a -p
may I suggest only having the long switch?